gef.py

```diff
@@ -6789,7 +6789,7 @@ def get_shellcode(self, sid: int) -> None:
         ok("Downloaded, written to disk...")
         tempdir = gef.config["gef.tempdir"]
         fd, fname = tempfile.mkstemp(suffix=".txt", prefix="sc-", text=True, dir=tempdir)
-        shellcode = res.splitlines()[7:-11]
+        shellcode = res.splitlines()[9:-1]
```

Comment:
I know that this PR does not introduce this hacky way of extracting the shell code from the web page, but shouldn't we instead extract the content of the html <pre> tag? so something like `shellcode = res.split("<pre>")[1]`...

Reply:
Yeah, `res.split("<pre>")[1].split("</pre>")[0]` will be better for sure.

Reply:
```suggestion
        shellcode = res.split("<pre>")[1].split("</pre>")[0]
```